ibis/backends/base/sql/__init__.py

```diff
@@ -270,7 +267,7 @@ def _register_in_memory_table(self, table_op):
 
     def _register_in_memory_tables(self, expr):
         if self.compiler.cheap_in_memory_tables:
-            for memtable in graph.traverse(_find_memtables, expr.op()):
+            for memtable in graph.traverse(an.find_memtables, expr.op()):
```

Comment:
ideally this should be:
```suggestion
            for memtable in an.find_memtables(expr.op()):
```